source/_docs/terminus/plugins.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Terminus Manage Plugins
+terminuspage: true
+type: terminuspage
+layout: terminuspage
+permalink: docs/terminus/:basename/
+---
+Extend the functionality of Terminus and add commands by installing third party plugins.
+## Install Plugins
+Add plugins within the `$HOME/.terminus/plugins` directory on your local workstation. You may need to create the `$HOME/.terminus/plugins` directory if it does not already exist.
```

Comment:
One of the installation steps in the "advanced instructions" should be deemed to be the easiest / most basic, and pulled outside of the collapsed div so it is always visible as the standard recommended installation technique. This method should also be used in all of the plugin README files.